
this is hack to make rk3328 corrcect colorf after adding aw 10bit/afbc support in
0535-dw-hdmi-add-10bit-and-afbc-support3.patch


diff --speed-large-files --no-dereference --minimal -Naur linux-6.5.8/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c linux-6.5.8/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c
--- linux-6.5.8/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c	2023-10-23 10:51:10.466682085 +0200
+++ linux-6.5.8/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c	2023-10-23 10:50:00.186681859 +0200
@@ -1057,16 +1057,16 @@
 		break;
 
 	case MEDIA_BUS_FMT_YUV8_1X24:
-		color_format = 0x17;
+		color_format = 0x09;
 		break;
 	case MEDIA_BUS_FMT_YUV10_1X30:
-		color_format = 0x18;
+		color_format = 0x0b;
 		break;
 	case MEDIA_BUS_FMT_YUV12_1X36:
-		color_format = 0x19;
+		color_format = 0x0d;
 		break;
 	case MEDIA_BUS_FMT_YUV16_1X48:
-		color_format = 0x1A;
+		color_format = 0x0f;
 		break;
 
 	default:
